overlord/ifacestate/ifacemgr.go

```diff
@@ -72,12 +78,31 @@ func Connect(s *state.State, plugSnap, plugName, slotSnap, slotName string) (*st
 	// TODO: Store the intent-to-connect in the state so that we automatically
 	// try to reconnect on reboot (reconnection can fail or can connect with
 	// different parameters so we cannot store the actual connection details).
-	summary := fmt.Sprintf(i18n.G("Connect %s:%s to %s:%s"),
+	plugHookSetup := &hookstate.HookSetup{
+		Snap:     plugSnap,
```

Comment:
Unless some sort of useful evidence shows up, we should probably kill the idea of running hooks of a given revision. So far we haven't made any use of it at all, so it's been just added complexity.